src/Surfnet/StepupRa/RaBundle/Controller/ProfileController.php

```diff
@@ -18,19 +18,27 @@
 
 namespace Surfnet\StepupRa\RaBundle\Controller;
 
+use Surfnet\StepupRa\RaBundle\Service\ProfileService;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
 final class ProfileController extends Controller
 {
     public function profileAction()
     {
-        $token  = $this->get('security.token_storage')->getToken();
         $logger = $this->get('logger');
 
         $logger->notice('Opening profile page');
 
-        $identity = $token->getUser();
-        $profile = $this->get('ra.service.profile')->findByIdentityId($identity->id);
+        $identity = $this->getUser();
```

Comment:
#boyscoutrule ?

Reply:
Yes, boyscouted. Because it didn't change any functionality I didn't bother to mention this.

Reply:
I'm a big fan of these improvements, but as reviewer find them rather confusing. Maybe you can tag your commit, containing the change with a boyscout mark/comment?

Reply:
I did already merge this PR back and there are already changes on develop. So  I haven't done this.